askomics/static/js/AskomicsGraphBuilder.js

```diff
@@ -436,22 +436,49 @@
       constraintRelations.push(["?"+'URI'+secondNode.SPARQLid,"?"+"id_start_"+secondNode.SPARQLid,"?"+startSecNodeId]);
       constraintRelations.push(["?"+'URI'+secondNode.SPARQLid,"?"+"id_end_"+secondNode.SPARQLid,"?"+endSecNodeId]);
 
-      var opstart = "";
-      var opend = "";
+      var id_link = secondNode.id+'-'+node.id;
+
+      type = $('#direction-'+id_link).val();
```

Comment:
it will be better to save the value of #direction inside the link (todo in the AskomicsLinkView.js) with a new attribute (link.direction ?) when the select change.
Firstly you dissociate the view from model. Secondly, we can store, inside a askomics request file, the value of the link and so regenerate the kind of link !